checkov/secrets/runner.py

```diff
@@ -319,6 +319,17 @@ def run(
                 self._modify_invalid_secrets_check_result_to_skipped(report)
             return report
 
+    @staticmethod
+    def _prioritise_secrets(secret_records: Dict[str, SecretsRecord], secret_key: str, check_id: str) -> bool:
```

Comment:
```suggestion
    def _should_prioritise_secrets(secret_records: Dict[str, SecretsRecord], secret_key: str, check_id: str) -> bool:
```
either rename or change the return type to not be a boolean